wordstats/language_data/t2l/code/README.md

```diff
@@ -0,0 +1,62 @@
+### cleaning_words.py
+
+If it is required to add words of a new language it is needed to follow the next steps:
+
+1. Download the aspell dictionary of that language (http://ftp.gnu.org/gnu/aspell/dict/) and follow the instructions. Basically you have to download the .tar.bz2 file (the 5 is the newest) and then:
+
+	-Unzip the folder
+
+	-Go with the terminal to that folder.
+
+	-Execute ./configure
+
+	-Execute make
+
+	-Execute make install
+
+	-Execute preunzip 'lang'.cwl (e.g: es.cwl)
+
+You will obtain a .wl file with the dictionnary.
+
+However there will be a script which will do this automatically, but if the result is not what you expect, then you can do it manually.
```

Comment:
there is a script now. 
so update the text.
but i like that you left also the manual instructions, in case the script does not work.